### PR TITLE
Change role display in message adherent filter

### PR DESCRIPTION
**Pull request by kevin-verschaeve**
[ch11418]

image

In the "exclude" select, the first row is disabled because it is selected in the "include" select

### CODE REVIEW

File: src/Entity/AdherentMessage/Filter/BasicUserFiltersTrait.php [hunk 1]
```diff
@@ -26,21 +26,21 @@ trait BasicUserFiltersTrait
      *
      * @ORM\Column(type="boolean")
      */
-    private $includeCommitteeSupervisors = true;
+    private $includeCommitteeSupervisors;
```
Comment (kevin-verschaeve) [inline src/Entity/AdherentMessage/Filter/BasicUserFiltersTrait.php:29]: the idea behind this change is : if true, add the filter, if false, add the filter but negationned, if null, just skip the filter. It allows us to not rely on a new field. WDYT ?

Comment (ottaviano) [reply src/Entity/AdherentMessage/Filter/BasicUserFiltersTrait.php:29]: yes, this is a good approach for me

File: src/Form/FilterRoleType.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class FilterRoleType extends AbstractType
+{
+    public const ROLES = [
+        'adherent_message.filter.role.committeeHosts' => 'CommitteeHosts',
+        'adherent_message.filter.role.committeeSupervisors' => 'CommitteeSupervisors',
+        'adherent_message.filter.role.citizenProjectHosts' => 'CitizenProjectHosts',
+    ];
```
Comment (Remg) [inline src/Form/FilterRoleType.php:15]: what about keeping this array "clean" with only values,
and add this in the `setDefaults` method:
```
'choice_label' => function (string $choice) {
    return "adherent_message.filter.role.$choice";
},
```

Comment (kevin-verschaeve) [reply src/Form/FilterRoleType.php:15]: done

**Pull request merged**